player/lua/ytdl_hook.lua

```diff
@@ -215,8 +215,8 @@ local function edl_track_joined(fragments, protocol, is_live, base)
 end
 
 local function has_native_dash_demuxer()
-    local demuxers = mp.get_property_native("demuxer-lavf-list")
-    for _,v in ipairs(demuxers) do
+    local demuxers = mp.get_property_native("demuxer-lavf-list", {})
```

Comment:
If this is aimed at helping people backport ytdl, you might want to include some of the information from the PR intro in the commit message.